drivers/soundwire/intel.c

```diff
@@ -1217,7 +1217,7 @@ static void *intel_get_sdw_stream(struct snd_soc_dai *dai,
 		dma = dai->capture_dma_data;
 
 	if (!dma)
-		return NULL;
+		return ERR_PTR(-EINVAL);
```

Comment:
The commit message says ``snd_soc_dai_get_sdw_stream() can only return -ENOTSUPP or the stream, NULL is not a possible value.``, but we return ``ERR_PTR(-EINVAL)`` here. Did I miss something?

Reply:
yes, the commit message is incorrect, should be 

"snd_soc_dai_get_sdw_stream() can only return the pointer to stream or an ERR_PTR value, NULL is not a possible value"

Reply:
fixed